src/main/resources/db/migration/V1_0__initial_schema.sql

```diff
@@ -46,7 +46,7 @@ CREATE TABLE `users` (
   `date_of_birth` datetime DEFAULT NULL,
   `picture` longblob,
   `status` varchar(255) DEFAULT NULL,
-  `username` varchar(255) NOT NULL,
+  `username` varchar(255) NOT NULL UNIQUE,
```

Comment:
Please revert this UNIQUE constraint and create a new script that adds it

Reply:
`ALTER TABLE user ADD CONSTRAINT username_unique UNIQUE (username);`

something like this